corehq/apps/sms/tests/migration.py

```diff
@@ -607,3 +619,134 @@ def testSQLSyncToCouch(self):
         couch_obj = ExpectedCallbackEventLog.get(sql_obj.couch_id)
         self.checkFieldValues(couch_obj, sql_obj, ExpectedCallback._migration_get_fields())
         self.assertTrue(ExpectedCallbackEventLog.get_db().get_rev(couch_obj._id).startswith('3-'))
+
+
+class PhoneNumberMigrationTestCase(BaseMigrationTestCase):
+    def getCouchCount(self):
+        result = VerifiedNumber.view(
+            'sms/verified_number_by_domain',
+            startkey=[self.domain],
+            endkey=[self.domain, {}],
+            include_docs=False,
+            reduce=True
+        ).all()
+        return result[0]['value'] if result else 0
+
+    def getSQLCount(self):
+        return PhoneNumber.objects.filter(domain=self.domain).count()
+
+    def setRandomCouchObjectValues(self, obj):
+        obj.domain = self.domain
+        obj.owner_doc_type = self.randomString()
+        obj.owner_id = self.randomString()
+        obj.phone_number = self.randomString()
+        obj.backend_id = self.randomString()
+        obj.ivr_backend_id = self.randomString()
+        obj.verified = self.randomBoolean()
+        obj.contact_last_modified = self.randomDateTime()
+
+    def setRandomSQLObjectValues(self, obj):
+        obj.domain = self.domain
+        owner_doc_type = self.randomString()
+        owner_id = self.randomString()
+        phone_number = self.randomString()
+        backend_id = self.randomString()
+        ivr_backend_id = self.randomString()
+        verified = self.randomBoolean()
+        contact_last_modified = self.randomDateTime()
+
+    def testCouchSyncToSQL(self):
+        self.deleteAllLogs()
+        self.assertEqual(self.getCouchCount(), 0)
+        self.assertEqual(self.getSQLCount(), 0)
+
+        # Test Create
+        couch_obj = VerifiedNumber()
+        self.setRandomCouchObjectValues(couch_obj)
+        couch_obj.save()
+
+        sleep(1)
```

Comment:
yeah i only see these issues locally, but for some reason if i don't wait the doc is not found